src/pages/VisiumQC.tsx

```diff
@@ -180,14 +182,15 @@ export default function VisiumQC() {
     }
     if (
       values.qcType === QCType.VISIUM_CONCENTRATION ||
+      values.qcType === QCType.QPCR_RESULTS ||
       (values.qcType === QCType.CDNA_AMPLIFICATION && values.slotMeasurements)
     ) {
       sendCDNA({
         type: 'SUBMIT_FORM',
         values: {
           workNumber: values.workNumber,
           barcode: values.barcode,
-          slotMeasurements: values.slotMeasurements ?? [],
+          slotMeasurements: values.slotMeasurements?.map(({ samples, ...rest }) => rest) ?? [],
```

Comment:
Are we getting rid of 'samples' field? if so, why?

Reply:
yes, because we don't need in the request, we are just adding some info in the display. also the GraphQL request will fail if we pass the wrong arguments .